### PR TITLE
[terra-divider] Accessibility guide update

### PR DESCRIPTION
**Pull request by sugan2416**
### Summary

Added accessibility guide for `terra-divider`

Closes #

### Deployment Link

[Deployment link](https://terra-core-divider-a11y-wusrnm.herokuapp.com/components/cerner-terra-core-docs/divider/accessibility-guide)

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-core-docs/src/terra-dev-site/doc/divider/AccessibilityGuide.4.doc.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import DividerExampleAccess from "./example/DividerExampleAccess";
```
Comment (saket2403) [inline packages/terra-core-docs/src/terra-dev-site/doc/divider/AccessibilityGuide.4.doc.mdx:1]:
```suggestion
import { Badge } from 'terra-divider/package.json?dev-site-package';
import DividerExampleAccess from "./example/DividerExampleAccess";

<Badge />
```

File: packages/terra-core-docs/src/terra-dev-site/doc/divider/AccessibilityGuide.4.doc.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+- Content creators must provide engineers with the desired heading text.
+  - Work with engineers to understand the content structure of the page to ensure heading levels for all Dividers are appropriate for their placement within the page.
+      - Dividers with text should never be a heading level 1.
+- Best practice dictates that headings should be used to identify sections of content.
+  - If a Divider is used with text, the Divider text will act as the appropriate heading.
+  - If no heading text is provided within the Divider, best practice would be to include a heading within the content after the divider.
+      - Dividers with no text should not have a heading level assigned to the Divider itself.
+
+
+
```
Comment (saket2403) [inline packages/terra-core-docs/src/terra-dev-site/doc/divider/AccessibilityGuide.4.doc.mdx:69]: nit: Extra lines

File: packages/terra-core-docs/src/terra-dev-site/doc/divider/AccessibilityGuide.4.doc.mdx (hunk 1 above)
Comment (sugan2416) [reply packages/terra-core-docs/src/terra-dev-site/doc/divider/AccessibilityGuide.4.doc.mdx:2]: added in https://github.com/cerner/terra-core/pull/3677/commits/90dbc52654717fe1c86bc3f5c6788289f093c9b3

File: packages/terra-core-docs/src/terra-dev-site/doc/divider/AccessibilityGuide.4.doc.mdx (hunk 2 above)
Comment (sugan2416) [reply packages/terra-core-docs/src/terra-dev-site/doc/divider/AccessibilityGuide.4.doc.mdx]: removed in https://github.com/cerner/terra-core/pull/3677/commits/90dbc52654717fe1c86bc3f5c6788289f093c9b3

**Review by saket2403 — APPROVED** [review]

**Review by supreethmr — APPROVED** [review]

**Pull request merged**